app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java

```diff
@@ -50,6 +54,8 @@
     private static final String ARGUMENT_HEIGHT = "height";
     private static final String ARGUMENT_OVERLAY = "translucent";
 
+    private String pasteString = "";
```

Comment:
@xfix You are right. This was an oversight on my end. I've now moved the initialization to just before the onCreateView